src/app/user-help/user-help.component.ts

```diff
@@ -0,0 +1,52 @@
+import { Component, OnInit } from '@angular/core';
+import { UserServiceService } from '../user-service.service';
+import {isArray} from 'util';
+
+@Component({
+  selector: 'app-user-help',
+  templateUrl: './user-help.component.html',
+  styleUrls: ['./user-help.component.css']
+})
+export class UserHelpComponent implements OnInit {
+
+  questions: Object[];
+  initialStep: boolean;
+  preserveAnswer: Object[];
+  showFinish: boolean;
+  showPrev: boolean;
+  showNext: boolean;
+
+  getQuestions(): void {
```

Comment:
Please, move all methods under the constructor.

Reply:
was applied